app/src/main/res/layout/no_collections_message.xml

```diff
@@ -7,6 +7,7 @@
     android:id="@+id/no_collections_wrapper"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:layout_marginHorizontal="@dimen/home_item_horizontal_margin"
```

Comment:
Is it safe to assume you went through each of the viewholder in SessionControlAdapter and found their respect xml and added the `android:layout_marginHorizontal` for each?

Reply:
Yes. First looked in the `SessionControlView` (reason for the commits split) then in `SessionControlAdapter`. Should've covered everything on homescreen.